services/drupal/scripts/ecs/drush-migrate.sh

```diff
@@ -42,13 +43,30 @@ run_migration() {
       --limit=$batch_size \
       --continue-on-failure \
       "$migration"
+
+    # Check if we've asked to halt. Migration's own status field isn't really reliable
+    # for discerning whether a migration has been stopped or completed.
+    halted=$(drush state-get epa.migrations_halted)
```

Comment:
Make sure you quote the capture here.
```suggestion
    halted="$(drush state-get epa.migrations_halted)"
```

Reply:
Ah because the captured string could have spaces...